### DISCUSSION TITLE
use kubectl  create a backup resource instead of velero

### DISCUSSION THREAD

**Discussion by Yelijah** [vmware-tanzu/velero#5363] [Community support Q&A] [answered]
I am a new guy for velero. 
Can i use kubectl cli and backup api mentioned by https://velero.io/docs/v1.9/api-types/  to do backup and restore instead of velero cli ?

**Comment by sseago** [comment, marked as answer]
Yes. The velero CLI is just a shortcut to creating a backup/restore CR. The end result of `velero backup create` is just creating a Backup CR. In some cases, it's easier to just create the CR directly.

**Reply by Yelijah** [reply]
Thank u for your answer!
